assets/app/index.html

```diff
@@ -5,8 +5,11 @@
     <base href="/">
     <title>OpenShift Web Console</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="images/apple-touch-icon-precomposed.png">
```

Comment:
Yes, it's a rename of the file, to what it's referenced as in the <head>
touch-icon-precomposed.png to apple-touch-icon-precomposed.png